src/neo/NeoSystem.cs

```diff
@@ -66,7 +65,7 @@ public NeoSystem(ProtocolSettings settings, string storageEngine = null, string
             this.TaskManager = ActorSystem.ActorOf(Network.P2P.TaskManager.Props(this));
             foreach (var plugin in Plugin.Plugins)
                 plugin.OnSystemLoaded(this);
-            Blockchain.Ask<FillCompleted>(new FillMemoryPool { Transactions = Enumerable.Empty<Transaction>() }).Wait();
+            Blockchain.Ask(GenesisBlock).Wait();
```

Comment:
Use a different message like `BlockchainInit`? we can avoid the condition for all blocks

Reply:
Done.